embulk-input-mysql/src/main/java/org/embulk/input/mysql/MySQLInputPlugin.java

```diff
@@ -172,7 +175,11 @@ private void loadTimeZoneMappings()
                 f.set(null, timeZoneMappings);
             }
         }
-        catch (ClassNotFoundException | IllegalAccessException | NoSuchFieldException | IOException e) {
+        catch (ClassNotFoundException e) {
+            // It appears that the user uses the Connector/J 8.x driver.
+            // Do nothing;
```

Comment:
Just in case, we may want to 
```suggestion
            try {
                Class.forName("com.mysql.cj.util.TimeUtil");
            } catch (fianl ClassNotFoundException ex2)
                // Throw if neither the Connector/J 5.x nor 8.x driver is found.
                throw new RuntimeException(ex);
            }
            // Pass-through if the Connector/J 8.x driver is found.
```

Reply:
Thanks! I found typos (`fianl` and missing `{`), I fixed without using `commit suggestion`.